website/content/api-docs/auth/aws.mdx

```diff
@@ -282,10 +284,21 @@ $ curl \
 ## Create Certificate Configuration
 
 Registers an AWS public key to be used to verify the instance identity
-documents. While the PKCS#7 signature of the identity documents have DSA
-digest, the identity signature will have RSA digest, and hence the public
-keys for each type varies respectively. Indicate the type of the public key
-using the "type" parameter.
+documents. Indicate the type of the public key using the "type" parameter.
```

Comment:
nit: I think we can probably format the parameter as code to be consistent in the other docs.
```suggestion
documents. Indicate the type of the public key using the `type` parameter.
```